crates/bevy_gltf/src/convert_coordinates.rs

```diff
@@ -1,11 +1,11 @@
-use core::f32::consts::PI;
+use std::f32::consts::PI;
```

Comment:
Why not core? An effort has been made to make as much as possible support no_std for portability.